### PR TITLE
Tree Shake | Image Render Priority | UI bug fixes

### PR DESCRIPTION
**Pull request by GabrielPedroza**
## Description

PR Accomplishment: Removed console logs, added priority to image (disable lazy loading), increase z-index for forms (ui bug)

Image priority:

Images won't be lazy loaded. Because images are important in this project, especially the landing page, images to be lazily rendered as it is by default should be removed. It is also LCP (Largest Contentful Paint) in my project so having it prioritized is the best move. More information on this: https://nextjs.org/docs/pages/api-reference/components/image#priority

Future PR Accomplishment: Future Polish the capstone project. Remove hardcode from forms.

## Test Plan

UI Bug issue (z-index): https://ibb.co/nDpH16b | Bug Fix: https://ibb.co/9sSQKN2

Image priority implementation still renders image: https://ibb.co/smxV5y0

Follow previous PR to set up SendGrid: https://github.com/GabrielPedroza/bread/pull/14 SG mail promise (error handling removed still sends email to end user): https://ibb.co/ysyKp7L

The reason for error handling removal is because if an error arises, there is no way for the user to know since there isn't a feature for execution logs.

### CODE REVIEW

**Comment by yangdanny97** [comment]
Screenshot 2023-08-04 at 14-51-19 orangutan interview Meme Generator - Imgflip

**Comment by yangdanny97** [comment]
Elaborate on the priority thing? Is it so the images load faster?

File: src/utils/sendEmail.ts [hunk 1]
```diff
@@ -16,13 +16,6 @@ export const sendEmail = () => {
...
-  sgMail
-    .send(msg)
-    .then(() => {
-      console.log("Email sent");
-    })
-    .catch((error) => {
-      console.error(error);
-    });
+  // eslint-disable-next-line @typescript-eslint/no-floating-promises
+  sgMail.send(msg)
```
Comment (yangdanny97) [inline src/utils/sendEmail.ts:20]: failed email send is unhandled? 

I guess you haven't really built out execution logs for these tasks so its OK for now

**Comment by GabrielPedroza** [comment]
Will have a PR template with test plan written soon

**Comment by GabrielPedroza** [comment]
> Elaborate on the priority thing? Is it so the images load faster?

Images won't be lazy loaded. Because images are important in this project, especially the landing page, I don't want the image to be lazily rendered as it would look weird. It is also a LCP in my project so having it prioritized is the best move. Will write about this in template. https://nextjs.org/docs/pages/api-reference/components/image#priority

File: src/utils/sendEmail.ts (hunk 1 above)
Comment (GabrielPedroza) [reply src/utils/sendEmail.ts:20]: > failed email send is unhandled?
> 
> I guess you haven't really built out execution logs for these tasks so its OK for now

Yeah execution logs aren't built out and showing the error in the UI is obsolete as the user won't be on the site (most probably) when the webhook is executed. Having this unhandled is probably the only option for now.

**Pull request merged**